.ci/jenkins/project/Jenkinsfile.post-release

```diff
@@ -89,9 +86,6 @@ pipeline {
         }
 
         stage('Update OptaPlanner website') {
```

Comment:
you would update the website with OP 8 ?
won't that create problems ?

Reply:
The two streams will create two PRs to the website; even if they come simultaneously, they should be automatically mergeable, because:
- the XSDs have the major version in filenames
- the ansible-playbook.yaml is going to change only for 9
- the pom.yml is going to be updated by both PRs, but each modifies a different section of it

That said, the next release to come will (dis/ap)prove my thinking.